dev/update_mlflow_versions.py

```diff
@@ -16,7 +16,8 @@ def replace_occurrences(files: List[Path], pattern: str, repl: str) -> None:
     pattern = re.compile(pattern)
     for f in files:
         old_text = f.read_text()
-        assert pattern.search(old_text), f"Pattern {pattern} not found in {f}"
```

Comment:
We're now searching over all Java files, most of which won't have the desired string. I don't think we strictly need the assert since we're manually reviewing the PRs anyway